Content/Projectiles/Ranged/Javelin/JavelinThrown.cs

```diff
@@ -2,30 +2,30 @@
 using Terraria.GameContent;
 using Terraria.ID;
 
-namespace PathOfTerraria.Content.Items.Gear.Weapons.Javelins;
+namespace PathOfTerraria.Content.Projectiles.Ranged.Javelin;
 
 public class JavelinThrown(string name, Vector2 itemSize, int dustType) : ModProjectile
 {
 	protected override bool CloneNewInstances => true;
-	public override string Name => name;
-	public override string Texture => $"{PathOfTerraria.ModName}/Assets/Items/Gear/Weapons/Javelins/{name.Replace("Thrown", "")}";
+	public override string Name => InstanceName;
+	public override string Texture => $"{PathOfTerraria.ModName}/Assets/Items/Gear/Weapons/Javelins/{InstanceName.Replace("Thrown", "")}";
 
-	public bool Stabbing
+	public bool UsingAlt
 	{
 		get => Projectile.ai[2] == 1;
 		set => Projectile.ai[2] = value ? 1 : 0;
 	}
 
-	private string name = name;
-	private Vector2 itemSize = itemSize;
-	private int dustType = dustType;
+	protected string InstanceName = name;
+	protected Vector2 ItemSize = itemSize;
+	protected int DustType = dustType;
 
 	public override ModProjectile Clone(Projectile newEntity)
 	{
 		var proj = base.Clone(newEntity) as JavelinThrown;
-		proj.name = name;
-		proj.itemSize = itemSize;
-		proj.dustType = dustType;
+		proj.InstanceName = InstanceName;
+		proj.ItemSize = ItemSize;
+		proj.DustType = DustType;
 		return proj;
 	}
```

Comment:
Can just add a setter to `Name` and get rid of `InstanceName`.

Reply:
Can't add a setter; `Name` doesn't provide one to override.

Reply:
I forgot you can't provide a setter for a property whose base class only defines a getter, seems like it should be a sensible language feature. That's fine then.